ingest/environment.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - click
   - ijson
   - jsonlines
-  - ncbi-datasets-cli >=16.21.0
+  - ncbi-datasets-cli >=16.29.0
```

Comment:
Just checked and replacing with ncbi-datasets-cli would also work and might be better - @corneliusroemer I assume you fixed this for a reason - are you ok with unfixing it?

Reply:
I'm not sure I understand what you mean here